src/cephadm/cephadm

```diff
@@ -91,15 +43,62 @@ from threading import Thread, RLock
 from urllib.error import HTTPError
 from urllib.request import urlopen
 
-cached_stdin = None
 
+# Default container images -----------------------------------------------------
+DEFAULT_IMAGE = 'docker.io/ceph/daemon-base:latest-master-devel'
+DEFAULT_IMAGE_IS_MASTER = True
```

Comment:
worth standardizing on ``'`` or ``"``?

Reply:
I'm in favor of that, personally I prefer `'` ..

Reply:
@liewegas https://github.com/ceph/ceph/pull/39699